ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2835,24 +2840,20 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
-- as: gcp-ipi-secureboot-confidential-computing-mini-perm-f14
-  cron: 18 23 8,24 * *
+- as: gcp-ipi-secureboot-confidential-mini-perm-f14
```

Comment:
Compared with `gcp-ipi-confidential-fips-mini-perm-f28-destructive`, the job enabled secureboot on purpose, can we rename the job to `gcp-ipi-confidential-secureboot-mini-perm-f14`, so that `gcp-ipi-confidential-fips-mini-perm-f28-destructive` and `gcp-ipi-confidential-secureboot-mini-perm-f14` can be placed together.